ext/mbstring/libmbfl/mbfl/mbfilter.h

```diff
@@ -104,6 +104,8 @@
 #define ssize_t __int64
 #elif defined(_WIN32)
 #define ssize_t __int32
+#elif defined(__GNUC__) && __GNUC__ >= 4
+#define ssize_t __int128
```

Comment:
Pretty sure `__int128` here is not correct. This should be either an int32 or int64 depending on platform, but not int128. As the Windows case is already handled an I think pretty much everything else is LP64 doing `#define ssize_t long` should do the trick.

Reply:
I will be fix it